src/njoy21/input/ACER/Card7/Thin3.hpp

```diff
@@ -0,0 +1,13 @@
+struct Thin3 {
+  using Value_t = int;
+  static std::string name() { return "thin3"; }
+
+  static std::string description(){
+    return 
+        "The thin3 (or iskf) argument, specifies the skip factor. Use every\n"
```

Comment:
gotta love the old school Fortran programmers, using gottfriend notation in their (english documentation)